### DISCUSSION TITLE
Where to get complied Android App

### DISCUSSION THREAD

**Discussion by sskelly** [OpenOrienteering/mapper#2298] [Questions & Answers] [answered]
This is probably a dumb question, but is there a compiled version of the Android app available for download somewhere? I realized it's not on the Play store, but I can't find any references anywhere to even direct download an apk for install.

**Comment by pkturner** [comment, marked as answer]
If you go to the [main page for Mapper](https://www.openorienteering.org/apps/mapper/), click on "Stable Release", and then follow the _second_ link labeled [More downloads for this version](https://github.com/OpenOrienteering/mapper/releases/tag/v0.9.5) it shows several compiled apk versions for Android.

**Reply by sskelly** [reply]
Wow, I completely missed that. I had clicked the link and thought I had been to a different page. Thanks.
